### ISSUE TITLE
Docx: Figure reference link not working

### ISSUE THREAD

**Issue by et2010** [lierdakil/pandoc-crossref#147]
Hi, 

I found reference link for figure doesn't work. But ref link for equation and table works as expected. I can click on a ref link for a table and the cursor will jump to the correct position. But for figure, it always jump to the beginning of the documents. I checked the xml soure and found there isn't any bookmark for figure. But I don't know how to fix it since I know little about haskell.

I'm using the latest pandoc along with pandoc-crossref 0.3 beta

**Comment by lierdakil** [comment]
Probably similar to https://github.com/lierdakil/pandoc-crossref/issues/105

вт, 28 нояб. 2017 г., 18:35 et2010 <notifications@github.com>:

**Comment by bokov** [comment]
Problem still remains in [v0.3.3.0](/lierdakil/pandoc-crossref/releases/download/v0.3.3.0/linux-ghc84-pandoc23.tar.gz).  As happened for OP, links to tables work for me, and everything works for HTML output suggesting I'm gettting the syntax right...

Pandoc info:

    pandoc 2.3
    Compiled with pandoc-types 1.17.5.1, texmath 0.11.1, skylighting 0.7.2

After a lot of trial and error I found the following possible work-around using [Subfigures](https://lierdakil.github.io/pandoc-crossref/#subfigures) syntax:

```
::::: {#fig:testdivfig custom-style="divfig"}

![](rmdtest_files/figure-html/divfig-1.png)<!-- -->

Overall caption

:::::

```
In short, I created a div with just one sub-figure and references to `[@fig:testdivfig]` correctly linked to that figure when generating Word output (the custom style is likely optional, just kept it in because it's handy to have).

**Comment by bokov** [comment]
No worries. I agree with you that it's an upstream problem.

Does it even make sense for pandoc-crossref to support direct image IDs? From an architecture standpoint it might even be more elegant to treat single figures as a special case of subfigures... but there may be use-cases of which I am not aware where they need image IDs.

As for me, though, the hack suits me fine.

**Comment by myutkin** [comment]
Hi lierdakil,

I was wondering if you had time to look into this?

**Comment by lierdakil** [comment]
Hi. Thanks for reminding me. I've had a bit of free time today, so I created a pull request fixing the issue on the pandoc repo: https://github.com/jgm/pandoc/pull/5054

**Comment by myutkin** [comment]
Thank you.
